docs/source/index.rst

```diff
@@ -25,7 +25,7 @@ SkyPilot is a framework for easily and cost effectively running ML workloads on
 
 SkyPilot abstracts away cloud infra burden:
 
-- Launch jobs & clusters on any cloud (AWS, Azure, GCP)
+- Launch jobs & clusters on any cloud (AWS, Azure, GCP, Lambda)
```

Comment:
Let's update this line in README.md too!

Reply:
Good point! I totally forgot about that.

Reply:
nit: Lambda Cloud?

Reply:
Updated!